program/verilator-verification.ppl.md

```diff
@@ -0,0 +1,33 @@
+# OpenHW Preliminary Project Proposal: Verilator verification for CORE-V MCU SoC
+
+
+[[_TOC_]]
+
+## SUMMARY OF PROJECT
+This project aims to provide a stable and reasonably reliable Verilator verification flow:
```

Comment:
I would use "Verilator" process, rather than "Verilator verification flow", since the SW side is not particularly about verification.